client/signup/steps/emails/index.jsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import EmailStepSideBar from 'calypso/components/emails/email-step-side-bar';
+import EmailSignupTitanCard from 'calypso/components/emails/email-signup-titan-card';
+import StepWrapper from 'calypso/signup/step-wrapper';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class EmailsStep extends React.Component {
+	//TODO
+	handleSkip = () => {};
+
+	renderSideContent = () => {
+		return (
+			<div className="emails__email-side-content-container">
+				<div className="emails__email-side-content">
+					<EmailStepSideBar />
+				</div>
+			</div>
+		);
+	};
+
+	renderContent() {
+		const content = (
+			<div className="register-email-step">
+				<CalypsoShoppingCartProvider>
+					<EmailSignupTitanCard
+						siteUrl={ this.props.progress.domains.siteUrl }
+						//TODO
+						addButtonTitle={ this.props.translate( 'Add' ) }
+						skipButtonTitle={ this.props.translate( 'Skip' ) }
+						onAddButtonClick={ () => {} }
+						onSkipButtonClick={ () => {} }
+						//TODO
+						price={ '3.5$' }
```

Comment:
I think we should be using the `getProductDisplayCost` selector with a product slug of `TITAN_MAIL_MONTHLY_SLUG`.

We may eventually switch to the `getProductCost` selector and then passing that through `formatCurrency`, but that is not needed for this first iteration.